packages/astro/src/events/session.ts

```diff
@@ -108,7 +107,7 @@ export function eventCliSession(
 					: undefined,
 				markdown: userConfig?.markdown
 					? {
-							mode: userConfig?.markdown?.mode,
```

Comment:
mode is no longer supported but the types were out of date, so this wasn't caught. Removed this and added `drafts` instead.